docs/usage/integration.md

```diff
@@ -35,7 +35,7 @@ What specific guarantees does TigerBeetle provide to applications?
 - Multiple client sessions may receive replies [out of order](#reply-order) relative to one another.
 - A client session can consider a request executed when it receives a reply for the request.
 
-#### [Requests](#requests)
+#### [Requests](#client-requests)
```

Comment:
Why do we have headers that are links to different sections? If a header is a link I would expect it is a link to itself. Though I wouldn't expect a header to be a link in the first place.

Reply:
In the initial draft the level-4 headers were list bullets, e.g.:

- Sessions
  - A client session may have at most one in-flight request.
  - A client session reads its own writes.
  - ...
- Requests
  - ...

I switched it to headers b/c I thought it was more readable. And thinking about it more, the links here probably aren't necessary at all; I can just remove them.